templates/s3/schema.capnp

```diff
@@ -2,24 +2,18 @@
 
 struct Schema {
     mgmtNic @0: Nic; # zerotier management nic
-    storageNic @1: Text; # vxlan management nic
-    farmerIyoOrg  @2: Text; # the farmer to create the s3 on
-    dataShards @3: Int32=16; # 0-stor data shards config
-    parityShards @4: Int32=4; # 0-stor parity shards config
-    storageType @5: StorageType; # s3 storage type
-    storageSize @6: UInt16; # s3 storage size
-    namespaces @7: List(Namespace); # namespace services created for s3. This is set by the template.
-    minioLogin @8: Text; # minio login
-    minioPassword @9: Text; # minio password
-    minioUrl @10: Text; # url to access minio on. This is set by the template.
-    gateway @11: Text; # Gateway service to use for vm
-    gatewayRobot @12: Text; # Robot that created the gaty
-    gatewayPublicNetwork @13: Text; # Gateway network to use for the vm
-    gatewayPrivateNetwork @14: Text; # Gateway network to use for the vm
-    nsPassword @15: Text; # Namespace password
-    vmMacaddress @16: Text; # vm macaddress retrieved from the gateway
-    vmIp @17: Text; # Vm ip retrieved from the gateway
-    vmPort @18: Int32; # Vm forward port retrieved from the gateway
+    farmerIyoOrg  @1: Text; # the farmer to create the s3 on
+    dataShards @2: Int32=16; # 0-stor data shards config
+    parityShards @3: Int32=4; # 0-stor parity shards config
+    storageType @4: StorageType; # s3 storage type
+    storageSize @5: UInt16; # s3 storage size
+    namespaces @6: List(Namespace); # namespace services created for s3. This is set by the template.
+    minioLogin @7: Text; # minio login
+    minioPassword @8: Text; # minio password
+    minioUrl @9: Text; # url to access minio on. This is set by the template.
+    nsPassword @10: Text; # Namespace password
+    vmIp @11: Text; # Vm ip retrieved from the gateway
```

Comment:
you don't need the vmIp and vmPort if you're not using the storageNic anymore.

Reply:
ok I'll remove